modules/nf-core/deeptools/alignmentsieve/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process DEEPTOOLS_ALIGNMENTSIEVE"
+    script "../main.nf"
+    process "DEEPTOOLS_ALIGNMENTSIEVE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "deeptools"
+    tag "deeptools/alignmentsieve"
+
+    test("homo_sampiens - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.sorted.bam.bai', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.logs,
+                                  process.out.versions)
+                                  .match()
```

Comment:
Try this for the bam file, it uses the [nft-bam plugin ](https://nvnieuwk.github.io/nft-bam/latest/):
```suggestion
                { assert snapshot(process.out.logs,
                                  bam(process.out.bam[0][1]).getReadsMD5(),
                                  process.out.versions)
                                  .match()
```